airbyte-server/src/main/java/io/airbyte/server/config/ApplicationBeanFactory.java

```diff
@@ -98,8 +98,8 @@ public JsonSchemaValidator jsonSchemaValidator() {
   }
 
   @Singleton
-  public AirbyteGithubStore airbyteGithubStore() {
-    return AirbyteGithubStore.production();
+  public AirbyteRemoteOssCatalog remoteOssCatalog() {
+    return new AirbyteRemoteOssCatalog();
   }
```

Comment:
I believe if there's no arguments to pass this can be removed as long as the AirbyteRemoteOssCatalog is marked as `@Singleton` itself